packages/runtime/src/next/components/SlotProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import { createContext, ReactNode } from "react";
+
+export const PropsContext = createContext<{
+    fallback: ReactNode
+}>({
+    fallback: null
+});
+
+function SlotProvider({ value, children }: { value: {fallback: ReactNode}, children: ReactNode }) {
+    return (
```

Comment:
Looks like we've got 4 spaces for indentation here. `pnpm format` should take care of it